lib/ui/compositing/scene_host.cc

```diff
@@ -48,30 +48,35 @@ flutter::SceneHost* GetSceneHost(scenic::ResourceId id) {
   return nullptr;
 }
 
-void InvokeDartClosure(tonic::DartPersistentValue* closure) {
-  if (closure) {
-    std::shared_ptr<tonic::DartState> dart_state = closure->dart_state().lock();
-    if (!dart_state) {
-      return;
-    }
-
-    tonic::DartState::Scope scope(dart_state);
-    tonic::DartInvoke(closure->value(), {});
+void InvokeDartClosure(const tonic::DartPersistentValue& closure) {
+  auto dart_handle = closure.value();
+  if (Dart_IsNull(dart_handle) || !Dart_IsClosure(dart_handle)) {
```

Comment:
If the handle is not a closure, that seems like it would indicate that Dart and native aren't talking to each other as intended. Should that be a CHECK or DCHECK instead of swallowing it?

Reply:
Haha, I was thinking this exact same thing while eating lunch.  Will make this a DCHECK